pkg/handlers/docker/mutation/customimage/inject_control_plane.go

```diff
@@ -29,8 +29,8 @@ import (
 )
 
 const (
-	// WorkerHandlerNamePatch is the name of the inject handler.
-	WorkerHandlerNamePatch = "DockerWorkerCustomImagePatch"
+	// ControlPlaneHandlerNamePatch is the name of the inject handler.
```

Comment:
Noticed some typos here. Because we use the meta handlers these don't actually change the behavior in a running controller.